src/Data/Expenses/Ledger.hs

```diff
@@ -111,41 +121,48 @@ showMoney (amount, currency) =
   let (dollars, cents) = properFraction amount
       dollars' = showCommaSeparatedNumber dollars
       cents' = printf "%.02d" (truncate $ cents * 100 :: Integer) :: String
-  in [i|#{dollars'}.#{cents'} #{currency}|]
+  in  [i|#{dollars'}.#{cents'} #{currency}|]
 
 
 
 showCommaSeparatedNumber :: Int -> String
 showCommaSeparatedNumber x | x < 1000 = show x
 showCommaSeparatedNumber x =
   let first = showCommaSeparatedNumber (x `div` 1000)
-      rest = printf "%03d" (x `mod` 1000) :: String
-  in [i|#{first},#{rest}|]
+      rest  = printf "%03d" (x `mod` 1000) :: String
+  in  [i|#{first},#{rest}|]
 
 
 
 showHumanReadableMoney :: (D.Decimal, String) -> String
 showHumanReadableMoney (amount, currency) =
-  let (dollars, cents) = properFraction amount :: (Integer, D.Decimal)
-      trailing3Zeros = dollars `mod` 1000 == 0 && cents == 0
-      useM = dollars > 1000000 && trailing3Zeros
-      useK = dollars > 1000 && (dollars `mod` 1000 == 0 || dollars `mod` 1000 >= 100) && cents == 0
-      (dollars', cents', modifier)
-        | useM =
-          ( dollars `div` 1000000
-          , D.Decimal 3 $ (dollars `mod` 1000000) `div` 1000
-          , "m"
-          )
-        | useK =
-            (dollars `div` 1000, D.Decimal 3 $ dollars `mod` 1000, "k")
-        | otherwise =
-            (dollars, cents, "")
-      humanReadableDollars = showCommaSeparatedNumber $ fromIntegral dollars'
-      humanReadableCents
-        | cents' == 0 = ""
-        | useM || useK = drop 1 $ show $ D.normalizeDecimal cents'
-        | otherwise = drop 1 $ show cents'
-  in [i|#{humanReadableDollars}#{humanReadableCents}#{modifier} #{currency}|]
+  let
+    (dollars, cents) = properFraction amount :: (Integer, D.Decimal)
+    trailing3Zeros   = dollars `mod` 1000 == 0 && cents == 0
+    useM             = dollars > 1000000 && trailing3Zeros
+    useK =
+      dollars
+        >  1000
```

Comment:
This is also not a good example, although I don't know how the formatter is supposed to know that.